test/controllers/personal_access_tokens_controller_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class PersonalAccessTokensControllerTest < ActionController::TestCase
+  let(:token) { FactoryGirl.create(:personal_access_token)  }
+  let(:user) { token.user }
+
+  test 'new' do
```

Comment:
you can use https://github.com/theforeman/foreman/blob/develop/test/controllers/shared/basic_rest_response_test.rb and replace some of the tests like at https://github.com/theforeman/foreman/blob/develop/test/controllers/architectures_controller_test.rb#L8